bundles/model/org.eclipse.smarthome.model.script/src/org/eclipse/smarthome/model/script/interpreter/ScriptInterpreter.xtend

```diff
@@ -66,12 +75,20 @@ public class ScriptInterpreter extends XbaseInterpreter {
 	override protected invokeFeature(JvmIdentifiableElement feature, XAbstractFeatureCall featureCall,
 		Object receiverObj, IEvaluationContext context, CancelIndicator indicator) {
 		if (feature != null && feature.eIsProxy) {
-			throw new RuntimeException(
-				"The name '" + featureCall.toString() + "' cannot be resolved to an item or type.");
+		    if (featureCall instanceof XMemberFeatureCall) {
+                throw new ScriptExecutionException(new ScriptError(
```

Comment:
https://www.emacswiki.org/pics/static/TabsSpacesBoth.png

Reply:
argh, good spot! No idea why there were tabs, but that's history now 😉

Reply:
https://github.com/glebm/render-whitespace-on-github/blob/master/README.md ;)

Reply:
Awesome, thanks for the hint! I started wondering already how you spotted them...

Reply:
Check out https://github.com/brunocvcunha/awesome-userscripts/blob/master/README.md#github

In combination with the above I can recommend https://github.com/jerone/UserScripts/tree/master/Github_Commit_Whitespace#readme to not loose focus